tests/testthat/test-dm-table-surgery.R

```diff
@@ -69,7 +69,12 @@ test_that("table surgery functions fail in the expected ways?", {
     data_ts_src,
     ~ expect_error(
       decompose_table(., aex_id, a, e, x),
-      "."
+      # FIXME: error class might be subject to future change?
```

Comment:
- Don't need a FIXME here -- if the test fails it will be obvious why
- Perhaps: `expect_error(..., ".", class = if (packageVersion(...)...) "tidyselect_...")` with suitable wrapping

Reply:
I'm not sure why we'd need the `"."`, since the default case `regexp = NULL` should cover this.